### PR TITLE
Prisoner role from /tg/

### PR DESCRIPTION
**Pull request by totallynotkatfood**
## About The Pull Request

title

## Why It's Good For The Game

terra is gonna kick my ass for all these mapchanges smh

## Changelog
:cl:
add: Adds prisoner role
/:cl:

### CODE REVIEW

**Comment by ZombiesVsPlants** [comment]
Nice

**Comment by TheFakeElon** [comment]
Traitor prisoners need uplinks, they don't spawn with PDAs and stuff. I'd suggest forcing either an implant or pen

File: austation/code/modules/jobs/job_types/prisoner.dm [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/datum/job/prisoner
+	title = "Prisoner"
+	flag = PRISONER
+	department_head = list("The Security Team")
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 2
```
Comment (TheFakeElon) [inline austation/code/modules/jobs/job_types/prisoner.dm:7]:
```suggestion
	total_positions = 0
```

**Comment by MCterra10** [comment]
map conflict. good luck.

**Comment by MCterra10** [comment]
also don't change maps other than austation reee

**Comment by totallynotkatfood** [comment]
but then there’ll be no prisoner spawner on every other map smh

**Comment by MCterra10** [comment]
Map merge your shit, Kat

**Comment by MCterra10** [comment]
https://tgstation13.org/wiki/Map_Merger

**Comment by totallynotkatfood** [comment]
and thus an end to mapmerging bullshittery

**Comment by MCterra10** [comment]
you'll have to edit jobcode or something to disable the job on maps other than austation

File: austation/code/modules/jobs/job_types/prisoner.dm [hunk 2]
```diff
@@ -19,5 +19,3 @@
 	uniform = /obj/item/clothing/under/rank/prisoner
 	shoes = /obj/item/clothing/shoes/sneakers/orange
 	id = /obj/item/card/id/prisoner
-	ears = /obj/item/radio/headset
-	belt = /obj/item/pda
```
Comment (TheFakeElon) [inline austation/code/modules/jobs/job_types/prisoner.dm]: Why?

Comment (totallynotkatfood) [reply austation/code/modules/jobs/job_types/prisoner.dm]: The headsets and pda spawned by default

Comment (totallynotkatfood) [reply austation/code/modules/jobs/job_types/prisoner.dm]: adding it is a bit unnecessary

**Comment by TheFakeElon** [comment]
Seems alright, just waiting on Terra’s decision :)

File: austation/code/game/area/Space_Station_13_areas.dm [hunk 3]
```diff
@@ -1,3 +1,7 @@
-/area/medical/plumbing
-	name = "Plumbing"
-	icon_state = "chem"
```
Comment (TheFakeElon) [inline austation/code/game/area/Space_Station_13_areas.dm:3]: why?

**Comment by TheFakeElon** [comment]
oops. my bad

**Comment by TheFakeElon** [comment]
Conflicts

**Comment by TheFakeElon** [comment]
eh, I think it's a nice challenge! escaping and all.

Although, what happens when there's no security

**Comment by MCterra10** [comment]
closing. it's just hooking too many files to be worth it. I'd recommend you try and get this merged on bee.

**Comment by Cooliecan** [comment]
> eh, I think it's a nice challenge! escaping and all.
> 
> Although, what happens when there's no security

Make the role only available when there is a warden active.

**Pull request closed without merging**